example/lib/app/app.dart

```diff
@@ -52,6 +52,16 @@ class _ChewieDemoState extends State<ChewieDemo> {
       videoPlayerController: _videoPlayerController1,
```

Comment:
Same code is used between line 148 and 183. You could use the same _chewieController.